utPlayback.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+import os
+import sys
+import subprocess
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(dir_path+"/../../../")
+
+class utPlaybackOnDevice():
+    """
+    UT Player class
+
+    """
+    def __init__(self, session, playbackTool="gstreamer", prerequisites="", log=None):
```

Comment:
`def __init__(self, session, playbackTool="gstreamer", prerequisites="", log=None):`

should be

`def __init__(self, device="dut", log=None):`

What prerequisites do you expect? You've already prepared the box, it has the stream and your ready to play?

utStartPlay - is this going to be different per platform? isn't it aways going to be gStreamer? That would come from the deviceConfig would it not?